doc/api/https.md

```diff
@@ -115,15 +115,20 @@ See [`http.listen()`][] for details.
 ## https.get(options[, callback])
 <!-- YAML
 added: v0.3.6
+changes:
+  - version: v7.5.0
+    pr-url: https://github.com/nodejs/node/pull/10638
+    description: The `options` parameter can be a WHATWG `URL` object.
 -->
-- `options` {Object | string} Accepts the same `options` as
+- `options` {Object | string | URL} Accepts the same `options` as
   [`https.request()`][], with the `method` always set to `GET`.
 - `callback` {Function}
 
 Like [`http.get()`][] but for HTTPS.
 
-`options` can be an object or a string. If `options` is a string, it is
-automatically parsed with [`url.parse()`][].
+`options` can be an object, a string, or a [`URL`][] object. If `options` is a string, it is
```

Comment:
The same: `` [`URL`] `` should be added to the link references at the end of the file